CMakeLists.txt

```
@@ -335,6 +335,8 @@ endif()
 
 configure_file(${Ginkgo_SOURCE_DIR}/cmake/ginkgo.pc.in
     ${Ginkgo_BINARY_DIR}/ginkgo.pc @ONLY)
+file(GENERATE OUTPUT ${Ginkgo_BINARY_DIR}/ginkgo.pc
+     INPUT ${Ginkgo_BINARY_DIR}/ginkgo.pc)
```

This looks like it would cause issues for multi-config generators, and if the two versions of ginkgo.pc don't match. CMake actually complains about that in my case.
```suggestion
configure_file(${Ginkgo_SOURCE_DIR}/cmake/ginkgo.pc.in
    ${Ginkgo_BINARY_DIR}/ginkgo.pc.in @ONLY)
file(GENERATE OUTPUT ${Ginkgo_BINARY_DIR}/$<CONFIG>/ginkgo.pc
     INPUT ${Ginkgo_BINARY_DIR}/ginkgo.pc.in)
```